udata/core/dataset/models.py

```diff
@@ -276,14 +271,12 @@ def __str__(self):
     }
 
     meta = {
-        'allow_inheritance': True,
         'indexes': [
             '-created_at',
             'slug',
-            'organization',
             'resources.id',
             'resources.urlhash',
-        ],
+        ] + db.Owned.meta['indexes'],
```

Comment:
I wonder if `db.Owned.INDEXES` is easier to understand. Here it looks to be dynamic.